sqrl-planner/src/main/java/com/datasqrl/engine/database/relational/ddl/PostgresDDLFactory.java

```diff
@@ -68,11 +86,22 @@ private static String toSql(String name, String sqlType, boolean nullable) {
     return sql.toString();
   }
 
+  @Override
   public CreateIndexDDL createIndex(IndexDefinition index) {
     List<String> columns = index.getColumnNames();
     return new CreateIndexDDL(index.getName(), index.getTableId(), columns, index.getType());
   }
 
+  public CreateNotifyTriggerDDL createNotify(String name, List<String> primaryKeys) {
+    return new CreateNotifyTriggerDDL(name, primaryKeys);
+  }
+
+  public Pair<ListenDDL, OnNotifyDDL> createNotifyHelperDDLs(String name, List<String> primaryKeys) {
+    ListenDDL listenDDL = new ListenDDL(name);
```

Comment:
We should migrate these to calcite AST tokens soon, but for everything all at once